python/ray/experimental/state/api.py

```diff
@@ -50,6 +50,26 @@
         data = list_nodes(address="localhost:8265")
     ```
 """
+import logging
+
+logger = logging.getLogger(__name__)
+STATE_OBS_ALPHA_FEEDBACK_MSG = [
+    "\n==========ALPHA PREVIEW, FEEDBACK NEEDED ===============",
+    "State Observability APIs is currently in Alpha-Preview. ",
+    "If you have any feedback, you could do so at either way as below:",
+    "  1. Comment on API specification: https://tinyurl.com/api-spec",
```

Comment:
are these urls accessible by the OSS community as well?

Reply:
1 is not, 2 is open to public, 3 is internal anyscale slack channel.

Reply:
yeah, we should make everything public.